src/otx/core/model/entity/anomaly.py

```diff
@@ -0,0 +1,74 @@
+"""Base Anomaly OTX model."""
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+from otx.core.data.entity.anomaly import (
+    AnomalyClassificationBatchPrediction,
+    AnomalyDetectionBatchPrediction,
+    AnomalySegmentationBatchPrediction,
+)
+from otx.core.model.entity.base import OVModel
+
+if TYPE_CHECKING:
+    from openvino.model_api.models import Model
+    from openvino.model_api.models.anomaly import AnomalyResult
+
+
+class OVAnomalyModel(OVModel):
```

Comment:
why do we need an additional ov model? What is the difference between ovmodel and exporter?